src/endpoints/files.js

```diff
@@ -8,26 +9,38 @@ class Files extends BaseExtend {
     this.endpoint = 'files'
   }
 
-  Create(body) {
-    let requestBody = body
+  Create(body, contentType = null) {
+    const additionalHeaders = {}
```

Comment:
We could do with a check here in the event that a node env doesn't specify the header
```
if (isNode() && contentType === null) throw error
```

Reply:
yeh thats a good idea 👍